### ISSUE TITLE
Add plain PET (not PETG) as a filament type

### ISSUE THREAD

**Issue by Gnuclear** [bambulab/BambuStudio#3065]
There are filaments that are made of PET, which are not exactly the same as PETG.
Some examples:
https://forward-am.com/material-portfolio/ultrafuse-filaments-for-fused-filaments-fabrication-fff/standard-filaments/ultrafuse-pet/
https://formfutura.com/product/high-precision-pet/
https://aurarum.com.au/product-category/3d-printer-filament/pet-filament/

**Comment by zbrozek** [comment]
Yes please. Another example: https://www.fusionfilaments.com/shop/category/ht-pet-23

But more generally, I should be able to type in anything I like rather than have to select from a limited drop down menu.
